documentation/book/con-custom-resources-example.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-custom-resources.adoc
+
+[id='con-custom-resources-example-{context}']
+= {Productname} custom resource example
+Each {Productname}-specific custom resource is defined within the specification of a CRD.
+
+To understand the relationship between a CRD and a custom resource, let's look at a sample of the CRD for a Kafka topic.
+
+.Kafka topic CRD
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: CustomResourceDefinition
+Metadata: <1>
+  name: kafkatopics.kafka.strimzi.io
+  labels:
+    app: strimzi
+Spec: <2>
+  group: kafka.strimzi.io
+  versions:
+    {ApiVersion}
+  scope: Namespaced
+  names:
+    # ...
+    plural: kafkatopics
+    shortNames:
+    - kt <3>
+  additionalPrinterColumns: <4>
+      # ...
+  Validation: <5>
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          properties:
+            partitions:
+              type: integer
+              minimum: 1
+            replicas:
+              type: integer
+              minimum: 1
+              maximum: 32767
+      # ...
+----
+<1> The metadata for the topic custom resource, its name and a label to identify the resource.
```

Comment:
This metadata is defining a name and a label for the CRD, not a CR instance.

Reply:
Updated to say CRD:
The metadata for the topic CRD, its name and a label to identify the CRD.